media-libs/netpbm/files/netpbm-10.86.21-test.patch

```diff
@@ -0,0 +1,10 @@
+`print ("got color map\n");` removed from `editor/pnmquantall` so remove from test
```

Comment:
have you sent this upstream ?

Reply:
Same as above, I will.

Reply:
It's been fixed by upstream in advanced *10.90.00*